common/models/workspace.js

```diff
@@ -183,6 +183,12 @@ module.exports = function(Workspace) {
       if (template.package) {
         template.package.name = packageName;
         template.package.description = description;
+        
+        var loopback = 'loopback';
+        if (loopback in options) {
+          template.package.dependencies[loopback] = options.loopback;
+        }
```

Comment:
In order to switch between loopback 2.x and 3.x, it's not enough to change `dependencies.loopback`. 2.x applications must have `loopback-datasource-juggler` as a top-level dependency too. OTOH, LoopBack 3.0 will finally have juggler as a regular dependency of loopback itself.

```
// loopback 2.x app
"dependencies": {
    "loopback": "^2.0.0",
    "loopback-datasource-juggler": "^2.0.0"
}

// loopback 3.x app
"dependencies": {
    "loopback": "^3.0.0"
}
```

I am proposing to revert this change here and change dependencies only inside tests.